### ISSUE TITLE
#-858825257------------- Buttons must have discernible text

### ISSUE THREAD

**Issue by khyatit13** [khyatit13/Coding-practice#479]
<strong>Source:</strong>`<button aria-label="" aria-live="polite" class="button-padding--shrunk icon-end-empty" type="button"><calcite-icon class="icon icon--start" aria-hidden="true" icon="ellipsis" scale="s"></calcite-icon></button>`
<strong>Element Location: </strong>["#menu-button-3b290a3d8914459ca5f67f35bbf25fe1","button"]
<strong>Impact: </strong>critical
<strong>failureSummary:</strong> Fix any of the following:
  Element does not have inner text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute
  Element's default semantics were not overridden with role="none" or role="presentation"Tags: [Ljava.lang.String;@7612116b
Screenshot:`C:\Users\khy12817\Desktop\Test.PNG`
